### PR TITLE
Add 1.15 changelog

### PR DESCRIPTION
**Pull request by Lyndon-Li**
Add 1.15 changelog

### CODE REVIEW

**Review by blackpiglet — APPROVED** [review]

File: changelogs/CHANGELOG-1.15.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
+## v1.14
```
Comment (ywk253100) [inline changelogs/CHANGELOG-1.15.md:1]: Should be v1.15

Comment (Lyndon-Li) [reply changelogs/CHANGELOG-1.15.md]: Done

File: changelogs/CHANGELOG-1.15.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+https://github.com/vmware-tanzu/velero/releases/tag/v1.15.0
+
+### Container Image
+`velero/velero:v1.15.0`
+
+### Documentation
+https://velero.io/docs/v1.15/
+
+### Upgrading
+https://velero.io/docs/v1.14/upgrade-to-1.15/
```
Comment (ywk253100) [inline changelogs/CHANGELOG-1.15.md:13]: Here also need to be v1.15

Comment (Lyndon-Li) [reply changelogs/CHANGELOG-1.15.md]: Done

**Review by ywk253100 — DISMISSED** [review]

File: changelogs/CHANGELOG-1.15.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+For more information, check the document https://velero.io/docs/v1.15/backup-repository-configuration/.  
+
+#### Performance improvements
+In 1.15, several performance related issues/enhancements are included, which makes significant performance improvements in specific scenarios:  
+- There was a memory leak of Velero server after plugin calls, now it is fixed, see issue https://github.com/vmware-tanzu/velero/issues/7925
+- The `client-burst/client-qps` parameters are automatically inherited to plugins, so that you can use the same velero server parameters to accelerate the plugin executions when large number of API server calls happen, see issue https://github.com/vmware-tanzu/velero/issues/7806
+- Maintenance of Kopia repository takes huge memory in scenarios that huge number of files have been backed up, Velero 1.15 has included the Kopia upstream enhancement to fix the problem, see issue https://github.com/vmware-tanzu/velero/issues/7510
+
+### Runtime and dependencies
+Golang runtime: v1.22.6
```
Comment (blackpiglet) [inline changelogs/CHANGELOG-1.15.md:58]: The Golang version should be v1.22.8.

Comment (Lyndon-Li) [reply changelogs/CHANGELOG-1.15.md]: Done

**Review by kaovilai — APPROVED** [review]

File: changelogs/CHANGELOG-1.15.md [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+### Documentation
+https://velero.io/docs/v1.15/
+
+### Upgrading
+https://velero.io/docs/v1.15/upgrade-to-1.15/
+
+### Highlights
+#### Data mover micro service
+Data transfer activities for CSI Snapshot Data Movement are moved from node-agent pods to dedicate backupPods or restorePods. This brings many benefits such as:  
+- This avoids to access volume data through host path, while host path access is privileged and may involve security escalations, which are concerned by users.
```
Comment (kaovilai) [inline changelogs/CHANGELOG-1.15.md:18]: Should we mention https://github.com/vmware-tanzu/velero/issues/8249 issue workarounds here?

**Review by shubham-pampattiwar — APPROVED** [review]

**Review by ywk253100 — APPROVED** [review]

**Pull request merged**